fastlane-plugin-wpmreleasetoolkit.gemspec

```diff
@@ -38,6 +38,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'rake', '>= 12.3', '< 14.0'
   spec.add_dependency 'rake-compiler', '~> 1.0'
   spec.add_dependency 'xcodeproj', '~> 1.22'
+  spec.add_dependency 'java-properties', '~> 0.3.0'
```

Comment:
@AliSoftware I added `java-properties` to the `gemspec` because there is Release Toolkit functionality that depends on it. Is that the correct place for it versus the `Gemfile`?

Reply:
Yes 👍 

The `.gemspec` defines the dependencies that are needed for the gem to _run_, i.e. that are used in its implementation; in other words the transitive dependencies that any client app (like WPiOS or whatnot) will need to bring along because `release-toolkit` need it.

The `Gemfile` is what's needed to run the code of the present repo (`release-toolkit`) locally.
In most cases, for repositories that are hosting code of ruby libraries (like `release-toolkit` is), the dependencies are _mostly_ the same to run the lib and its code locally vs to use the lib as a dependency in another repo, hence why most of their `Gemfile` just contain the keyword `gemspec` in them, to tell ruby to import all the same dependencies that are already declared in the `.gemspec` when using the lib in local development. But sometimes local development might want some additional dependencies that are not needed when using the lib in another repo, but are useful for development, like `rubocop` or `rspec` to check the lib's code and run its tests when developing it.

Reply:
Thank you for the explanation! I haven’t needed to add a new dependency to the Release Toolkit before, so I wasn’t sure. 🙂